homebrew/Brewfile

```diff
@@ -0,0 +1,16 @@
+tap "homebrew/bundle"
+tap "homebrew/core"
+
+tap "wix/brew"
+tap "facebook/fb"
+
+# Play, record, convert, and stream audio and video
+brew "ffmpeg"
+
+# Apple simulator utilities
+brew "wix/brew/applesimutils"
```

Comment:
doesn't look like related change. let's move to a separate pr

Reply:
applesimutils are required for this PR to work, so I think a way to describe such dependency fits as part of PR. I've separated this into a separate commit.